src/Arcus.BackgroundJobs.CloudEvents/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -138,7 +138,7 @@ public static class IServiceCollectionExtensions
         /// </summary>
         /// <param name="services">The services collection to add the job to.</param>
         /// <param name="serviceBusNamespace">The Service Bus namespace to connect to. This is likely to be similar to <c>{yournamespace}.servicebus.windows.net</c>.</param>
-        /// <param name="topicName">The name of the Azure Service Bus Topic to process.</param>
+        /// <param name="topicName">The name of the Azure Service Bus Topic to process the received event.</param>
```

Comment:
I believe it is the name of the topic that contains the messages that we must proces ?
```suggestion
        /// <param name="topicName">The name of the Azure Service Bus Topic that contains the messages that must be processed.</param>
```
?